readthedocs/rtd_tests/tests/test_subprojects.py

```diff
@@ -77,16 +77,27 @@ def setUp(self):
                                        self.owner], main_language_project=None)
                 self.pip.add_subproject(self.subproject)
                 self.pip.translations.add(self.translation)
-
-    @override_settings(PRODUCTION_DOMAIN='readthedocs.org')
-    def test_resolver_subproject_alias(self):
         relation = self.pip.subprojects.first()
         relation.alias = 'sub_alias'
         relation.save()
+        get(Project, slug='sub_alias', language='ya')
+
+
+    @override_settings(PRODUCTION_DOMAIN='readthedocs.org')
+    def test_resolver_subproject_alias(self):
         with override_settings(USE_SUBDOMAIN=False):
             resp = self.client.get('/docs/pip/projects/sub_alias/')
             self.assertEqual(resp.status_code, 302)
             self.assertEqual(
                 resp._headers['location'][1],
                 'http://readthedocs.org/docs/pip/projects/sub_alias/ja/latest/'
             )
+
+    def test_resolver_subproject_subdomain_alias(self):
```

Comment:
This test causes `test_resolver_subproject_alias` to fail if it runs first ("fixed" by creatively renaming so it runs after).
I had a bit of a look around to see if there's a cache I can easily clean but couldn't find anything.
Depends how much of an issue you think this is, but if you're worried & can give me a pointer I'll happily update it to dtrt